### ISSUE TITLE
[mu bug] In the documentation, please indicate which function is bound to the `z` key

### ISSUE THREAD

**Issue by jpggithub** [djcb/mu#2564]
On https://www.djcbsoftware.nl/code/mu/mu4e/MSGV-Detaching-and-reattaching.html, the key `z` do a certain thing.
The documentation could indicate which function is bound to this key ?

I have mu4e 1.8.14 (Debian testing) and the actual release is 1.10.7.

**Comment by djcb** [comment]
1. It's documented in the manual, but we don't generally have all the function names in the manual... and not really planning doing that, too much work, sorry. `C-h k z` is your friend.

**Comment by jpggithub** [comment]
Thank you for your answer but `C-h k z` gives me : "z runs the command ignore (found in mu4e-view-mode-map), which is an interactive native-compiled Lisp function in ‘subr.el’." 
Not so useful, at least with the 1.8.14 version. I will try with the version >= 1.10.7  when Debian will update mu4e.

**Comment by djcb** [comment]
Yes, that's expected on 1.8.x: the function was introduced in 1.10.x.
